README.md

```diff
@@ -206,7 +207,7 @@ Credit to [Gregoire Vella on Behance](https://www.behance.net/gregoirevella) for
 
 ## 📝 License <a name="license"></a>
 
-This project is [MIT](.//https://github.com/Gleennkar/Budget-App-RoR/edit/Dev/LICENSE) licensed.
+This project is [MIT](.https://github.com/Gleennkar/Budget-App-RoR/tree/Project-setup) licensed.
 
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/221356086-940376e2-be19-42dd-aff4-3cf709c0d886.png)


- [ ] Great work on adding a License file but I guess while adding you break your link in your readme file shown in the image above :arrow_up: I don't see any clickable link in your LICENSE section :thinking: Kindly fix it to make your readme file descriptive and professional :+1: